app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_parts = command.split(" ")
+    if len(command_parts) != 3 or command_parts != "mv":
```

Comment:
The condition `command_parts != 'mv'` is incorrect. You should check if the first element of `command_parts` is 'mv', i.e., `command_parts[0] != 'mv'`.